docs/README.md

```diff
@@ -39,7 +39,8 @@ Other useful resources:
 
 ## Tools
 
-The default build of the docs is performed with Apache Maven.
+The default build of the docs is performed with Apache Maven, which requires an
+installation of Java.
```

Comment:
It also needs docker.. so maybe we should rearrange this section 

docker (for just authoring)
Jdk and Maven (for full build and validation)

Reply:
Reworked as suggested.